contrib/platform/test/com/sun/jna/platform/win32/COM/ShellApplicationWindowsTest.java

```diff
@@ -0,0 +1,232 @@
+package com.sun.jna.platform.win32.COM;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+import com.sun.jna.Native;
+import com.sun.jna.NativeLibrary;
+import com.sun.jna.platform.win32.OleAuto;
+import com.sun.jna.platform.win32.Variant;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WinDef.LONG;
+import com.sun.jna.platform.win32.WinDef.LONGLONG;
+
+import junit.framework.TestCase;
+
+public class ShellApplicationWindowsTest extends TestCase {
+
+	@Override
+	protected void setUp() throws Exception {
+		String[] commands = {"cmd", "/c", "start", "iexplore.exe","-nohome", "\"http://www.srh.noaa.gov/jax\""};
+		Runtime.getRuntime().exec(commands);
+		Thread.sleep(30000);
```

Comment:
Can we do something a little smarter than a 30 second wait and a browse to a specific website? Maybe launch it via Win32 API and `ShellExecute`?